README.md

```diff
@@ -152,6 +152,16 @@ To select an `ENIConfig` based upon availability zone set this to `failure-domai
 
 ---
 
+`AWS_VPC_ENI_MTU` 
```

Comment:
Is this option cluster scoped against node scope?

Reply:
All nodes in the cluster will get the same setting.